app/src/pages/inside/settingsPage/settingsTabs/defectTypesTab/modals/deleteModal/deleteModal.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { ModalLayout, withModal } from 'components/main/modal';
+import Parser from 'html-react-parser';
+import classNames from 'classnames/bind';
+import styles from './deleteModal.scss';
+
+const cx = classNames.bind(styles);
+
+@withModal('defectTypeDeleteModal')
+export class DeleteModal extends Component {
```

Comment:
Can it be done using `app/src/pages/inside/common/modals/deleteItemsModal` instead?